src/widget/widget.cpp

```diff
@@ -1447,6 +1447,7 @@ void Widget::searchContacts()
 
     contactListWidget->hide();
     contactListWidget->show();
+    contactListWidget->resize(QSize());
```

Comment:
structure:
contactListWidget->hide();
contactListWidget->show();

It is found in the file widget.cpp three times, can make a separate function Update ()?